boards/arm/stm32h745i_disco/board.cmake

```diff
@@ -0,0 +1,8 @@
+# STM32H745I-Disco board configuration
+# Copyright (c) 2022 Systerel (https://www.systerel.fr)
+# SPDX-License-Identifier: Apache-2.0
+
+board_runner_args(jlink "--device=STM32H745XI" "--speed=4000")
+
```

Comment:
I managed to get the blinky working with the modifications mentioned previously and then using cube programmer as a west flash runner:
```
board_runner_args(stm32cubeprogrammer "--erase" "--port=swd" "--reset=hw")
include(${ZEPHYR_BASE}/boards/common/stm32cubeprogrammer.board.cmake)
```